x-pack/plugins/security_solution/public/detections/components/rules/severity_mapping/index.tsx

```diff
@@ -15,14 +14,23 @@ import {
   EuiIcon,
   EuiSpacer,
 } from '@elastic/eui';
-import React, { useCallback, useMemo, useState } from 'react';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
 import styled from 'styled-components';
 import * as i18n from './translations';
 import { FieldHook } from '../../../../../../../../src/plugins/es_ui_shared/static/forms/hook_form_lib';
 import { SeverityOptionItem } from '../step_about_rule/data';
 import { CommonUseField } from '../../../../cases/components/create';
 import { AboutStepSeverity } from '../../../pages/detection_engine/rules/types';
+import {
+  IFieldType,
+  IIndexPattern,
+} from '../../../../../../../../src/plugins/data/common/index_patterns';
+import { FieldComponent } from '../../../../common/components/autocomplete/field';
+import { AutocompleteFieldMatchComponent } from '../../../../common/components/autocomplete/field_value_match';
 
+const SeverityMappingParentContainer = styled(EuiFlexItem)`
+  max-width: 471px;
```

Comment:
That is a specific number!

Reply:
I want to work with @marrasherrier after the first BC to determine the best styling for the serverity/risk overrides, as it's a lot of fields bunched up together (and is hard to make sense of the longer field/values). This is just to lock the container in place with the left-most portion of the form row.